amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2498,36 +2498,38 @@ paths:
           schema:
             $ref: '#/definitions/Error'
   ##retention-schedule
-  '/record-categories/{recordCategoryId}/retention-schedules':
+  '/record-categories/{recordCategoryId}/retention-schedule':
     post:
       tags:
         - retention-schedule
       summary: Create a retention schedule
       description: |
-        Create a retention schedule.        
+        Create a retention schedule.
+        
         For example, using the following JSON body will create a retention schedule:
         ```JSON
           {
-              "authority":"Retention Authority",
-              "instructions":"Retention Instructions",
-              "isRecordLevel":false
+              "authority": "Retention Authority",
+              "instructions": "Retention Instructions",
+              "isRecordLevel": false
           }
         ```
+
       operationId: createRetentionSchedule
       parameters:
         - $ref: '#/parameters/recordCategoryIdParam'
         - in: body
           name: nodeBodyCreate
```

Comment:
I'm not completely sure if this is used or not, but you could also change it for clarity to e.g. `retentionNodeBodyCreate`. Have a look what makes sense here.

Reply:
done the changes as per the comment.